Formula/pastebinit.rb

```diff
@@ -19,6 +11,7 @@ def install
     inreplace "pastebinit", "/usr/bin/python3", Formula["python3"].opt_bin + "python3"
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
+    rm_r etc/"pastebin.d"
```

Comment:
So should I only add fomlula caveat to warn user need to replace `pastebin.d/pastebin.com.conf` by themselves without delete `pastebin.d`?